### PR TITLE
feat: json pipe

### PR DESCRIPTION
**Pull request by PatrickJS**

### CODE REVIEW

**Comment by mgonto** [comment]
Thanks :).

Can't we merge the factory and pipe into one class and have it return itself on the `create` ?

**Comment by mgonto** [comment]
That would simplify code greatly :D. Also, let's return all the pipes from another file not from the `json.js`

**Comment by PatrickJS** [comment]
@mgonto yo, check it

File: src/utils/pipes/json.js [hunk 1]
```diff
@@ -1,9 +1,18 @@
...
+  supports(obj):boolean {
+    return true;
+  }
+
+  transform(value):string {
+    return JSON.stringify(value, null, 2);
+  }
+
+  create(cdRef):Pipe {
+    return new JSONPipeFactory(cdRef);
```
Comment (mgonto) [inline src/utils/pipes/json.js]: can't we just return this :)? We're not using the constructor anyway and we avoid creating another object

**Comment by PatrickJS** [comment]
@mgonto check yo

**Comment by mgonto** [comment]
You're my hero :heart:

**Pull request merged**